src/Symfony/Bundle/FrameworkBundle/Resources/config/session.xml

```diff
@@ -70,6 +70,8 @@
                 <argument key="session" type="service" id="session" on-invalid="ignore" />
                 <argument key="initialized_session" type="service" id="session" on-invalid="ignore_uninitialized" />
             </argument>
+            <argument>%kernel.debug%</argument>
```

Comment:
Is just here to avoid BC Break in Symfony 5 which has optional second debug parameter.